### ISSUE TITLE
BHL import; wrong image resolution setting

### ISSUE THREAD

**Issue by myrmoteras** [gsautter/goldengate-imagine#285]
Importing BHL content using ABBYY produces erroneous products that cannot be opened properly in GGI.

How to solve:
1. Open the file obtained from the file selector tool in BHL in ABBYY
2. In ABBYY 14, use image editor and set the resolution to arbitrary 400 dpi
3. Change to OCR mode
4. OCR
5. Save to PDF with text underneath
6. Open PDF in GGI as born digital and the glyphs only version
7. Run GGI

question to @gsautter how to find out the image resolution and size, or pixel size of an image in a scanned PDF?

**Comment by gsautter** [comment]
This is a pretty nasty flaw in BHL's excerpt generator, as I might have explained a couple of times before - and a pretty amateurish one, to be honest. Said generator basically loses the original page dimensions (specified in the source PDF) and substitutes the bitmap dimensions of the page image proper.

The combination of the page image bitmap dimensions and the PDF page dimensions normally facilitates computing page image resolution, but not so after the above foolacy (foolish fallacy ;-) , which basically results in trash data whose meaning is impossible to figure out without either a lot of handiwork or some quite rough automated guesses and assumptions.

Bottom line: GGI does make an effort in the aforementioned guesswork department, but there is no guarantee at all for GGI to be able to fully reconstruct the lost dimensional information.
